DocumentString.java

```diff
@@ -5,15 +5,15 @@
 /**
  * This class is thread safe.
  */
-public class ReadString implements Parsable {
-  private File file;
+public class DocumentString implements Document {
```

Comment:
@snrdrum could you please elaborate on naming? What is motivation to name it `DocumentString`?